web/src/tippyjs.ts

```diff
@@ -400,7 +400,7 @@ export function initialize(): void {
     });
 
     delegate("body", {
-        target: "#generate_multiuse_invite_radio_container.disabled_setting_tooltip",
+        target: "[data-tab-key='invite-link-tab'].disabled-tab",
         content: $t({
             defaultMessage:
                 "You do not have permissions to generate invite links in this organization.",
```

Comment:
For consistency, let's make it "You do not have permissions to create invite links in this organization."

Reply:
Done